kin-recovery/src/main/java/kin/recovery/backup/presenter/BackupPresenterImpl.java

```diff
@@ -20,13 +21,16 @@
 	private @Step
 	int step;
 	private final CallbackManager callbackManager;
+	private final KinAccount kinAccount;
 	private boolean isBackupSucceed = false;
 	private String accountKey;
 
 
-	public BackupPresenterImpl(CallbackManager callbackManager, @Nullable final Bundle savedInstanceState) {
+	public BackupPresenterImpl(CallbackManager callbackManager, @NonNull KinAccount kinAccount,
```

Comment:
`@NonNull` can be added to `callbackManager`

Reply:
will added it